src/components/Main/index.tsx

```diff
@@ -20,26 +21,60 @@ type Word = {
 const Main = () => {
   const randomWord = () => Math.floor(Math.random() * data.length)
   const [word, setWord] = useState<Word>(data[randomWord()])
+  const [voice, setVoice] = useState<SpeechSynthesisVoice | null>(null)
+
+  useEffect(() => {
+    const voices = window.speechSynthesis?.getVoices()
+    const jpVoice = voices?.find((voice) => /ja-JP/.test(voice.lang))
+
+    setVoice(jpVoice || voices?.[0])
+  }, [])
+
+  const speak = (word: string) => {
+    const patchedWord = word?.split('-').join('')
```

Comment:
Instead of getting romanji, why not read directly from Hiragana?

```suggestion
    const patchedWord = word
```